controller_manager:
  ros__parameters:
    update_rate: 50

    hardware:
      type: "arduino_car_hardware/ArduinoCarHardware"
      ros__parameters:
        serial_device: "/dev/ttyACM0"
        baud_rate: 115200
        timeout_ms: 100

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

    bicycle_controller:
      type: "bicycle_controller/BicycleController"
